src/server/editorServices.ts

```diff
@@ -16,6 +16,7 @@ namespace ts.server {
     }
 
     var lineCollectionCapacity = 4;
+    var cachePath = ts.combinePaths(process.env.HOME, ".typingCache");

```

Comment:
we should make this consistent across hosts as much as we can so that if you switch from VS/VSCode/Sublime on the same box you're using the same cache.  Perhaps just use process.env.home here and we can append "Microsoft/TypingsCache" in the downloadTypingFilesForJs